doc/tutorials/transformer/te_init.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>"We will be using a generic prediction UDF script. To execute queries and load data from<strong> the </strong>Exasol database<strong>, </strong>we will be using the <code><a href="https://github.com/exasol/pyexasol" target="_blank">pyexasol</a></code> module. "</p><p> </p><p>Also. Thsi file is kind of har to find as a starting point. maybe add a readme in this folder that points to it? or even move this notebook into the parent folder? and add a readme there? </p><p>also probably a mention of the different tutorial groups in the main readme of the repo would be nice. or possibly in the user guide. just to make them easier to find.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>"We will be using a generic prediction UDF script.". This is a copy/paste bug. Well spotted!</p>

Reply:
<p>also, there is a typo in the link to the config notebook and it does not work</p>

Reply:
<p>@ahsimb at some point we will need a notebook with a table of content, please create a ticket for that.</p>

Reply:
<p>Created</p>